packages/inquirer/lib/prompts/rawlist.js

```diff
@@ -146,6 +150,35 @@ class RawListPrompt extends Base {
 
     this.render();
   }
+
+  /**
+   * When user press up key
+   */
+
+  onUpKey() {
+    if (this.selected) {
+      this.selected--;
+    } else {
+      this.selected = this.opt.choices.realLength - 1;
+    }
+    this.rl.line = (this.selected + 1).toString();
```

Comment:
Is reassigning the `rl.line` necessary? I don't think it'll be in this case.

Reply:
I think the `rl.line` value should correspond to the selected index, otherwise it is not very strange?